lib/calendar.js

```diff
@@ -0,0 +1,62 @@
+import Simulator from './simulator-xcode-6';
+import path from 'path';
+import { fs, mkdirp } from 'appium-support';
+import { execSQLiteQuery } from './utils';
+
+class Calendar {
```

Comment:
i've seen this pattern and it's not a bad one, but we've typically stayed away from classes unless there's a real good reason to maintain state, and this class doesn't have any state at all. i'm not sure we need it just as a namespace. maybe simple functions would do just as well?